### DISCUSSION TITLE
Extension activation failed: "unable to get local issuer certificate"

### DISCUSSION THREAD

**Discussion by SomeshThakur** [community/community#7696] [Copilot Conversations] [answered]
How to fix this error?

`GitHub Copilot could not connect to server. Extension activation failed: "unable to get local issuer certificate"`

![image](https://user-images.githubusercontent.com/12843168/141685366-f8b55a8e-0c5c-45c0-8099-2474c34448b8.png)

**Comment by davecheney** [comment, marked as answer]
Hello, I'm sorry you are having difficulties. We've found that the copilot icon may not show up if copilot is not able to contact our servers. Are you behind a firewall, vpn, or other technology that may be intercepting your network connections?

**Reply by taoliujun** [reply]
Yes. i worked behind company's vpn which cross chinese firewall. how can i visit copilot behind vpn. Thks!!!

**Reply by davecheney** [reply]
Copilot doesn’t currently support socks or http proxies. If your vpn can operate at layer 3 you might have some luck but if something is MITMing your HTTPS traffic there isn’t much we can do. Sorry.

**Reply by imloualvaro** [reply]
Got same issue and im not using vpn, proxies. Trying to use with OSX BigSur + Virtual Studio Code 1.64.2

**Reply by davecheney** [reply]
Is it possible that your employer, internet service provider, or your government is interfering with your SSL traffic?

**Reply by anchal66** [reply]
Same issue I am facing. Till when can it be fixed?

**Reply by GabrielDornelas** [reply]
Same here, still waiting a fix.

**Comment by davecheney** [comment]
Duplicate of https://github.com/github/feedback/discussions/8866
